src/ORM/DataObject.php

```diff
@@ -2910,12 +2905,8 @@ public function isChanged($fieldName = null, $changeLevel = DataObject::CHANGE_S
     /**
      * Set the value of the field
      * Called by {@link __set()} and any setFieldName() methods you might create.
-     *
-     * @param string $fieldName Name of the field
-     * @param mixed $val New field value
-     * @return $this
      */
-    public function setField($fieldName, $val)
+    public function setField(string $fieldName, mixed $value): static
```

Comment:
Didn't really need to do `setField` but I was doing `getField` and `hasField` so it made sense to do this at the same time.

Changed  property name is to be consistent with the parent class naming so that named arguments can be used more reliably. That will be a theme with some of these.